Avo.js

```diff
@@ -170,13 +169,13 @@
   };
 
   InternalAvoLogger = {
-    logEventSent: function logEventSent(eventName, eventProperties, userProperties) {
-      const message = "Event Sent:" + eventName + "Event Props:" + JSON.stringify(eventProperties) + "User Props:" + JSON.stringify(userProperties);
+    logEventSent: function logEventSent(eventName, eventProperties, userProperties, groupProperties) {
+      const message = "Event Sent:" + eventName + "Event Props:" + JSON.stringify(eventProperties) + "User Props:" + JSON.stringify(userProperties) + "Group Props:" + JSON.stringify(groupProperties);
```

Comment:
why are group properties coming in here now, something that was forgotten?

Reply:
We haven't pulled Avo in this project since we launched Group properties 🚀